packages/sources/react/src/index.ts

```diff
@@ -31,6 +31,8 @@ export { VtmnProgressbar } from './components/indicators/VtmnProgressbar';
 export type { VtmnProgressbarProps } from './components/indicators/VtmnProgressbar';
 
 // Navigation
+export { VtmnSearch } from './components/navigation/search/VtmnSearch';
+export type { VtmnSearchProps } from './components/navigation/search/VtmnSearch';
```

Comment:
Can you please rename the "search" folder to "VtmnSearch" as we did with other components please? :)